### PR TITLE
Update index.mdx Sellers Guide Update

### PR DESCRIPTION
**Pull request by smteague**
Updating content under "I have an O+ contract"

### CODE REVIEW

File: src/content/resources/index.mdx [hunk 1]
```diff
@@ -21,7 +21,20 @@ To see a comprehensive overview of OASIS+ Apparent Contractor Awardees across th
...
-| Training - proposal submission (coming soon) | Learn how to submit your proposal. |
+|[FedConnect](http://FedConnect.net) | For managing your OASIS+ contract |
+|  |   **FedConnect - [Ready, Set, Go!](https://www.fedconnect.net/FedConnect/Marketing/Documents/FedConnect_Ready_Set_Go.pdf) - FedConnect Registration Guide |
+|  |   **FedConnect - [OASIS+ Guide](/sellers-guide/make-changes-contract/) - Request contract changes |
+|[VSC Registration](https://vsc.gsa.gov/vsc/app-registration) | Vendor Support Center (VSC) for access to eBuy. |
+| |VSC Phone: [1-877-495-4849](1-877-495-4849) / VSC Email: vendor.support@gsa.gov |
+|GSA eBuy | For seeing OASIS+ order opportunities (after registration in the VSC)|
+|   |  **[eBuy Homepage](https://www.ebuy.gsa.gov/ebuy/) | 
+|   |  **[eBuy Seller Guide](https://www.ebuy.gsa.gov/ebuy/assets/content/eBuy-Contractor_jobaid.pdf) |
```
Comment (weiwang-gsa) [inline src/content/resources/index.mdx:31]: [suggest to specify file type and size with it](https://designsystem.digital.gov/components/link/). e.g. [eBuy Seller Guide [PDF, 22 pages]

File: src/content/sellers-guide/index.mdx [hunk 2]
```diff
@@ -40,14 +40,21 @@ Initial proposal submissions: Closed on October 20, 2023.
         img_src="images/sellers/have-a-contract-image.svg"
         img_alt="Person standing amidst a series of charts and graphs."    
     >
-        ### [Confirm you’re registered in Symphony and administer your master contract](/sellers-guide/register-in-symphony/)  
-        Authorized employees must register in Symphony to complete administrative duties; check out the registration instructions to help you through the process.
+        ### [Register in FedConnect to manage your contract - FedConnect.net](https://www.fedconnect.net/FedConnect/Default.htm)
+        - Add “[CALM@unisonglobal.com](CALM@unisonglobal.com)” to your trusted address book so messages from your GSA Contracting team do not go to spam.
```
Comment (weiwang-gsa) [inline src/content/sellers-guide/index.mdx:44]: currently the link cause 404 error, email link needs to add "mailto:",  e.g.`[CALM@unisonglobal.com](mailto:CALM@unisonglobal.com)`

File: src/content/sellers-guide/index.mdx [hunk 3]
```diff
@@ -40,14 +40,21 @@ Initial proposal submissions: Closed on October 20, 2023.
...
+       ### [Register in the Vendor Support Center (VSC) to see offers](https://vsc.gsa.gov/vsc/app-registration) 
+       - All offers for orders against your OASIS+ master contract will be in GSA eBuy. 
+       - You must register your OASIS+ contracts in VSC prior to using eBuy. Register your contracts at [https://vsc.gsa.gov/vsc/app-registration](https://vsc.gsa.gov/vsc/app-registration). 
+       - After successful registration, you should be able to log in to GSA eBuy when ordering opens. You can log into eBuy at that time using this link: https://www.ebuy.gsa.gov/ebuy/ . 
+       - VSC registration will be open. Check your award letter for additional details and dates. 
 
-        ### [Report sales in the FAS Sales Reporting Portal (SRP)](/sellers-guide/reporting) 
-        Review reporting requirements and discover how to use the SRP.
+        ### [Register in the Sales Reporting Portal (SRP)](https://srp.fas.gsa.gov) 
+       - Register in SRP to report task order awards, report sales monthly (including “no sales” against awarded task orders), and remit collected Contract Access Fees (CAF) quarterly. [https://srp.fas.gsa.gov](https://srp.fas.gsa.gov)
```
Comment (weiwang-gsa) [inline src/content/sellers-guide/index.mdx:53]: We usually put link associate with text, e.g. `[Register in SRP]((https://srp.fas.gsa.gov) `, not sure if this meets the content style guide by wrapping link with link,  maybe @rosamundtgov will have more input regarding this.

**Review by weiwang-gsa — CHANGES_REQUESTED** [review]
Hi Stephanie, it looks good, just a few places that may need a little bit change. I also tag Rosamund for content style guide suggestion as I am not sure about that, thank you!

File: src/content/sellers-guide/index.mdx (hunk 2 above)
Comment (smteague) [reply src/content/sellers-guide/index.mdx:44]: thanks... added

File: src/content/sellers-guide/index.mdx (hunk 3 above)
Comment (smteague) [reply src/content/sellers-guide/index.mdx:53]: I will change that...thank you...

File: src/content/resources/index.mdx (hunk 1 above)
Comment (smteague) [reply src/content/resources/index.mdx:31]: corrected

**Comment by weiwang-gsa** [comment]
there are still two other places that are using the whole URL as link text, maybe one use "register your OASIS+ contracts in VSC" as link text, and the other use  "log in to GSA eBuy" as link text?
image

**Review by weiwang-gsa — APPROVED** [review]
Looks great to me!

**Pull request merged**